src/orders/components/OrderChangeWarehouseDialog/OrderChangeWarehouseDialog.tsx

```diff
@@ -60,8 +62,15 @@ export const OrderChangeWarehouseDialog: React.FC<OrderChangeWarehouseDialogProp
 
   const [query, setQuery] = React.useState<string>("");
   const [selectedWarehouseId, setSelectedWarehouseId] = React.useState<string>(
-    currentWarehouse?.id
+    null
   );
+
+  React.useEffect(() => {
+    if (currentWarehouse?.id) {
+      setSelectedWarehouseId(currentWarehouse?.id);
```

Comment:
```suggestion
      setSelectedWarehouseId(currentWarehouse.id);
```
I think the `id` is accessible inside the if, right?